source/Calamari.Tests/AWS/CloudFormation/CloudFormationFixture.cs

```diff
@@ -12,7 +12,7 @@ namespace Calamari.Tests.AWS.CloudFormation
     [Category(TestCategory.RunOnceOnWindowsAndLinux)]
     public class CloudFormationFixture
     {
-        string GenerateStackName() => $"calamariteststack{Guid.NewGuid().ToString("N").ToLowerInvariant()}";
+        string GenerateStackName() => $"calamaricloudformation{Guid.NewGuid():N}";
```

Comment:
Does it need to be lowercase? just wondering if this is one of those things in was that has some weird requirement?

Reply:
I was having all kinds of weird issues with this not validating correctly. Apparently it does support `-`, but didn't work 🤷